cmd/api/config.go

```diff
@@ -54,13 +54,15 @@ type config struct {
 			Enabled   bool `default:"true"`
 			KeepFiles int  `default:"5"` // number of files to keep
 		}
+		URL string `default:""`
 	}
 	TelemetryPublisher struct {
 		Enabled            bool   `default:"false"`
 		MetricsHubURL      string `default:""`
 		MetricsHubAPIKey   string `default:""`
 		PublishingInterval string `default:"10s"`
 	}
+	FromBackup bool `default:"false" env:"FROM_BACKUP"`
```

Comment:
> I've started with only one config, much like `BoostrapFromSnapshotURL`, but the problem with that is:
> 
> we can't commit a configured value in `config.json` for that config, because that would mean that in every deployment the validator would be bootstrapped from backup. Or the user would have to change the `config.json`, which is bad because `config.json` is committed.
> 
> This implies that the user would have to provide the URL every time he wanted to bootstrap a validator from a backup file. I thought that it is just bad UX. So I aimed for something simpler like: `FROM_BACKUP=true make testnet-up`.
>
> Does that make sense to you?

Yes, that's what I suspected with my comment:
> I think you landed on the current configuration since you wanted a way to have a true and false UX with an "opinionated" preconfigured URL at the config.json.

I'm not sure about asking them to set the URL to be bad UX. My view on this is that it is more like "acceptable" or even "transparent" UX.

We should remember that we're aiming to build a decentralized network, where validators shouldn't be strictly/blindly trusting anyone about the state of the network. When a validator syncs using a backup, they're indirectly trusting the source that it has built the state correctly. (Unless we have a way to validate snapshots if Tableland state roots are posted on-chain, which we might have in the future but not today)

To be clear, I think it's fine to leave the pre-configured URL. I'm just dropping some extra thoughts about how a default URL can bake trust assumptions. In any case, validators should know what it means to boostrap from a backup even if the validator has preconfigured values. My rambling here is mostly philosophical and semantics. :)

> I agree that grouping them is much better. This would lead to something like:

:+1: 

> (side note: I'm insisting on the word backup, instead of snapshot, just to be consistent with the code and ultimately with SQLIte Backup API)

Coo, no strong opinion here. I usually use "snapshot" since it's usually the word used in many other validators out there. 
I think "backup" sounds somewhat "personal", and "snapshot" is more agnostic. Since we have a "backup" we need this extra logic when loading it to "remove data" to make it agnostic. Again, maybe a delicatessen here but I'm fine with backup.

Reply:
Makes sense. I haven't looked for that angle. Also, it would probably make it easier for us debugging since we could choose a backup from a specific timestamp. 

Changing to:
`BOOTSTRAP_BACKUP_URL=https://storage.googleapis.com/tableland-staging/backups/tbl_backup_latest.db.zst make staging-up`